kafka-rest/src/main/java/io/confluent/kafkarest/v2/KafkaConsumerState.java

```diff
@@ -381,7 +381,7 @@ synchronized ConsumerRecord<KafkaKeyT, KafkaValueT> next() {
    * be invoked with the lock held, i.e. after startRead().
    */
   private synchronized void getOrCreateConsumerRecords() {
-    ConsumerRecords<KafkaKeyT, KafkaValueT> polledRecords = consumer.poll(0);
+    ConsumerRecords<KafkaKeyT, KafkaValueT> polledRecords = consumer.poll(Duration.ZERO);
```

Comment:
The only difference between these two methods is that `poll(long)` will block when the consumer group is rebalancing. It's worth checking that the code doesn't depend on this blocking in some way if you haven't already.